.changeset/glorious-turtles-try.md

```diff
@@ -0,0 +1,22 @@
+---
+"@fluidframework/test-end-to-end-tests": major
+"@fluidframework/container-runtime": major
+"@fluid-internal/test-service-load": major
+---
+
+Ability to enable grouped batching
+
+The `IContainerRuntimeOptions.enableGroupedBatching` option has been added to the container runtime layer and is off by default. This option will group all batch messages
+under a new "grouped" message to be sent to the service. Upon receiving this new "grouped" message, the batch messages will be extracted and given
+the sequence number of the parent "grouped" message.
+
+Upon enabling this option, if any issues arise, use the `Fluid.ContainerRuntime.DisableGroupedBatching` feature flag to disable at runtime. This option should **ONLY** be enabled after ensuring 99.9% of sessions contains these changes.
```

Comment:
Maybe clarify - "you can observe that 99.9% of your application sessions are on this or later version of FF" or something like that, otherwise not clear what sessions it refers to.